src/styles/main.scss

```diff
@@ -0,0 +1,86 @@
+$day-size: 100px;
+$border-color: #000;
+$backgroundColor: #eee;
```

Comment:
Variable names should be written in kebab-case. Please change `$backgroundColor` to `$background-color` to adhere to the code style rules.